worlds/witness/rules.py

```diff
@@ -225,24 +228,26 @@ def _has_item(item: str, world: "WitnessWorld", player: int,
     if item in player_logic.REFERENCE_LOGIC.ALL_REGIONS_BY_NAME:
         region = world.get_region(item)
         return region.can_reach
-    if item == "7 Lasers":
+    elif item == "7 Lasers":
```

Comment:
Why turn all of these into elif, if every block has a `return` anyway?

Reply:
True